src/System.Net.Requests/src/System/Net/ValidationHelper.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace System.Net
+{
+    internal class ValidationHelper
+    {
+        internal static string HashString(object objectValue)
+        {
+            return LoggingHash.HashString(objectValue);
```

Comment:
In other places we're using this directly. Please add a TODO to remove this class in the future.